bin/spec

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This was only added for the go binaries since it was top level. Moving it into a folder it was no longer needed. If we want the global spec that can be added separately